### ISSUE TITLE
Liens multiples etherpad_mypads_ynh

### ISSUE THREAD

**Issue by JocelynDelalande** [YunoHost-Apps/etherpad_mypads_ynh#95]
Après avoir fait une upgrade (vers la version testing de l'app) je me retrouve avec plusieurs liens mypads :

![Screenshot_2020-04-07 Etherpad Mypads](https://user-images.githubusercontent.com/429633/78723695-3e5c9c00-792c-11ea-8650-23b66307324e.png)

**Comment by JocelynDelalande** [comment]
Funny fact, every upgrade seem to add one link. Kinda bling-bling versioning ? ;-)

**Comment by maniackcrudelis** [comment]
That way you would be sure to never missed the link :grin: 
Were you already on testing before this upgrade ? (especially were you already on 1.8.0 ?) This testing is supposed to fix that issue.

**Comment by JocelynDelalande** [comment]
No, I was on latest `master`, then upgraded (`ynh app upgrade -u`) to `testing`, then upgraded (`ynh app upgrade -u`) to `mypads_ynh` branch (#83 ), and then back (`ynh app upgrade -u`) to `testing` The latest was just to toy around with this bug. 

I did not account acurately, maybe I did a few more upgrades.

Let me know if there is a useful upgrade path for me to test for you.

**Comment by maniackcrudelis** [comment]
As I guess it's a production instance, I won't ask you to start again ;)
But, now that I know what you did, I'll try to do the same to reproduce.

**Comment by JocelynDelalande** [comment]
> As I guess it's a production instance, I won't ask you to start again ;)
> But, now that I know what you did, I'll try to do the same to reproduce.

No, I would not have done that wild upgrade testing stuff on a production instance. So if you need to confirm something go ahead and ask :)

**Comment by maniackcrudelis** [comment]
I'm currently going on the same path as you did.

Could you start again as well, run each upgrade with --debug, and check each time the main page to see how many links do you have.
When the links are duplicated, it's interesting to have a look to the log.

**Comment by maniackcrudelis** [comment]
So, from master to testing, everything went fine. I still have only one link.
From testing to #83, I do have a duplicate. But that branch does not have the fix from testing. It will be ok when the PR will be merged to testing.

**Comment by JocelynDelalande** [comment]
OK, thanks, so it gets an explanation, and you probably reproduced it ! So I'll test testing once the PR will get merged. We let open until then ?

Thanks for your quick feedbacks <3.

**Comment by maniackcrudelis** [comment]
Yep why not keeping it open until we're sure it's solved.
